### PR TITLE
Malins-project-clue

### PR DESCRIPTION
**Pull request by malin798**

### CODE REVIEW

File: code/clue.js [hunk 1]
```diff
@@ -1,59 +1,246 @@
...
   age: 45,
   image: 'assets/green.png',
-  occupation: 'Entrepreneur'
-}
+  occupation: 'Reverend',
+  },
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+ {suspect: "Professor Plum", 
+  firstName: 'Peter',
```
Comment (ericaglimsholt) [inline code/clue.js]: Would be great to choose `'` or `"` instead of multiple

File: code/clue.js [hunk 2]
```diff
@@ -1,59 +1,246 @@
...
+  {name: "The Dining Room",
+  location: "East Wing",
+  color: "coral",
+  image: "https://images.pexels.com/photos/2504911/pexels-photo-2504911.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260"
+  },
+  {name: "The Conservatory",
+  location: "West/South Wing",
+  color: "orchid",
+  image: "https://images.unsplash.com/photo-1524070960420-47602852d30f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=675&q=80"
+  },
```
Comment (ericaglimsholt) [inline code/clue.js]: Suggestion on structure:
```
{
  name: "The Conservatory",
  location: "West/South Wing",
  color: "orchid",
  image: "https://images.unsplash.com/photo-1524070960420-47602852d30f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=675&q=80"
},
```

File: code/clue.js [hunk 3]
```diff
@@ -1,59 +1,246 @@
...
+  killerImage.src = killer.image;
+  killerImage.classList.add("reveal-styling");
+
+  document.getElementById("killerCard").classList.add("flip-cards");
+  document.getElementById('killerCardBack').style.background = killer.color;
+  document.getElementById("killerAge").innerHTML = `Age: ${killer.age}`;
+  document.getElementById("killerOccupation").innerHTML = `Occupation: ${killer.occupation}`;
+  document.getElementById('killerName').innerHTML = `${killer.firstName} ${killer.lastName}`;
+
+  if (killer.suspect === "Mrs White") {
```
Comment (ericaglimsholt) [inline code/clue.js:197]: Why just Mrs White?

File: code/index.html [hunk 4]
```diff
@@ -5,33 +5,71 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
+
+    <script src="https://cdn.jsdelivr.net/gh/mathusummut/confetti.js/confetti.min.js"></script><script>confetti.start()</script>
+
+    <link href="https://fonts.googleapis.com/css?family=Jim+Nightshade&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Teko&display=swap" rel="stylesheet">
```
Comment (ericaglimsholt) [inline code/index.html]: You can actually choose multiple fonts in Google Fonts so you only get one link!

File: code/index.html [hunk 5]
```diff
@@ -5,33 +5,71 @@
...
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
+
+      <article class="deck rotate-card">
+        <div class="card-inner" id="killerCard">
+
+          <div class="card-front">
+            <p class="card-title"  id="killerTitle"><img src="assets/header-image.png" width= 80px></img>KILLER</p>
```
Comment (ericaglimsholt) [inline code/index.html]: Try to add styling in the css instead of the html :)

**Review by ericaglimsholt — APPROVED** [review]
Good work on your project!
I really like the confetti. It was a nice surprise. What I would like to see be change is the cards... You can see the info for a second then it turns over so it's impossible to see what you have 
image

**Comment by malin798** [comment]
> Good work on your project!
> I really like the confetti. It was a nice surprise. What I would like to see be change is the cards... You can see the info for a second then it turns over so it's impossible to see what you have

It works perfectly on Chrome on WIndows.. I will have to take a look at it in another browser then, thanks! :)

**Comment by ericaglimsholt** [comment]
> > Good work on your project!
> > I really like the confetti. It was a nice surprise. What I would like to see be change is the cards... You can see the info for a second then it turns over so it's impossible to see what you have
> 
> It works perfectly on Chrome on WIndows.. I will have to take a look at it in another browser then, thanks! :)

Things not working the same way in every browsers is a struggle, sadly. My tip is to check how well things are doing in different browsers (and versions) here: https://caniuse.com/

File: code/clue.js (hunk 3 above)
Comment (malin798) [reply code/clue.js:197]: She has a white background and you cant see the textboxes very well since they are also white :)

**Pull request closed without merging**